Open-Source/Job-Boards/Readme.md

```diff
@@ -7,4 +7,4 @@
 - [Open Source Engineer](https://www.indeed.com/q-Open-Source-Engineer-jobs.html?vjk=1475599e24cc62aa) - Indeed
 - [Open Source Engineer](https://www.aquasec.com/about-us/careers/co/engineering/81.E1B/open-source-engineer/all/) - Aquasec
 - [RedHat Careers](https://www.redhat.com/en/jobs)
-- [Software Engineer - Open Source](https://jobs.apple.com/en-us/details/200385788/software-engineer-open-source) - Apple
+- [Software Engineer - Open Source](https://jobs.apple.com/en-us/search?search=Software%20Engineer&sort=relevance&location=united-states-USA) - Apple
```

Comment:
please make the location settings to worldwide and not just united states

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Brittle

Reply:
Done! +resovled the merge conflict.